examples/01_plotting/plot_carpet.py

```diff
@@ -59,11 +58,8 @@
 atlas_data = atlas_img.get_fdata()
 discrete_version = np.argmax(atlas_data, axis=3) + 1
 discrete_version[np.max(atlas_data, axis=3) == 0] = 0
-discrete_atlas_img = nib.Nifti1Image(
-    discrete_version,
-    atlas_img.affine,
-    atlas_img.header,
-)
```

Comment:
not related to this PR but is there an atlas fetcher we could use here instead of building one? seems to complicate the example and it's not related to plotting